ui/pages/confirmations/hooks/useConfirmationAlerts.ts

```diff
@@ -46,6 +47,7 @@ function useTransactionAlerts(): Alert[] {
   const resimulationAlert = useResimulationAlert();
   const signingOrSubmittingAlerts = useSigningOrSubmittingAlerts();
   const trustSignalAlerts = useTrustSignalAlerts();
+  const originTrustSignalAlerts = useOriginTrustSignalAlerts();
```

Comment:
I have missed this on the initial trustSignalAlers, but these are not transaction specific. We should not have them under the `useTransactionAlerts`. We should add them in the `useConfirmationAlerts`.